src/net.cpp

```diff
@@ -651,14 +651,14 @@ bool CNode::ReceiveMsgBytes(Span<const uint8_t> msg_bytes, bool& complete)
             bool reject_message{false};
             CNetMessage msg = m_deserializer->GetMessage(time, reject_message);
             if (reject_message) {
-                // Message deserialization failed.  Drop the message but don't disconnect the peer.
+                // Message deserialization failed. Drop the message but don't disconnect the peer.
                 // store the size of the corrupt message
                 mapRecvBytesPerMsgType.at(NET_MESSAGE_TYPE_OTHER) += msg.m_raw_message_size;
                 continue;
             }
 
-            // Store received bytes per message command
-            // to prevent a memory DOS, only allow valid commands
+            // Store received bytes per message type
+            // to prevent a memory DOS, only allow valid messages
```

Comment:
I think this should say "known message types"?

For example, a `reject` message might be valid according to BIP 61, but unknown to this version of Bitcoin Core.

Reply:
Thanks for seeing it. I think you are right. Using "known message types" would be a better wording for this sentence.
Let me address it by updating the second commit.